### PR TITLE
Update enable-attack-surface-reduction.md

### PR DESCRIPTION
**Pull request by cunninghamp**
Added clarification for where to find the OMA-URI value to use for the example ASR rule.

This was not immediately clear that the reader needed to reference another part of the page for the specific value to use when following the example.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ You can use Microsoft Endpoint Manager (MEM) OMA-URI to configure custom ASR rul
 
    - In **Name**, type a name for the rule.
    - In **Description**, type a brief description.
-   - In **OMA-URI**, type or paste the specific OMA-URI link for the rule that you are adding.
+   - In **OMA-URI**, type or paste the specific OMA-URI link for the rule that you are adding. Refer to the [MDM section of this page](https://docs.microsoft.com/en-gb/microsoft-365/security/defender-endpoint/enable-attack-surface-reduction?view=o365-worldwide#mdm) for the OMA-URI to use for this example rule.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md]:
```suggestion
   - In **OMA-URI**, type or paste the specific OMA-URI link for the rule that you are adding. Refer to the MDM section earlier in this topic for the OMA-URI to use for this example rule.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cunninghamp, kindly commit the suggested change. Many thanks.

File: microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md (hunk 1 above)
Comment (jweston-1) [reply microsoft-365/security/defender-endpoint/enable-attack-surface-reduction.md]:
```suggestion
  - In **OMA-URI**, type or paste the specific OMA-URI link for the rule that you are adding. Refer to the MEM section earlier in this topic for the OMA-URI to use for this example rule. For ASR rule GUIDS, see [Per rule descriptions](attack-surface-reduction-rules.md#per-rule-descriptions) in the topic: Attack surface reduction rules.
```

**Pull request merged**